README.md

```diff
@@ -14,10 +14,13 @@ The project operates on an open contribution model, welcoming contributions from
 To see what's incoming, check the open [pull requests](https://github.com/greenelab/deep-review/pulls).
 For project discussion and planning see the [Issues](https://github.com/greenelab/deep-review/issues).
 
-### Current stage: resubmitted to _JRSI_ and _bioRxiv_
+### Current stage: planning Deep Review 2019
 
 We [completed](https://github.com/greenelab/deep-review/releases/tag/v0.10) manuscript revisions in [response](content/response-to-reviewers.md) to two external peer reviews.
-Please see [issue #810](https://github.com/greenelab/deep-review/issues/810) for a discussion about future plans and whether we will continue to accept new pull requests.
```

Comment:
You could remove line 19 about `We completed manuscript revisions` if this section is about the next release.

Reply:
This is a good idea. Thanks.